RNTester/RNTester.xcodeproj/project.pbxproj

```diff
@@ -3715,6 +3715,10 @@
 		13B07F941A680F5B00A75B9A /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ARCHS = (
```

Comment:
Can we also make sure this works with Cocoapods enabled?

Reply:
Yep. I suspect its going to be a change to the new Cocoapod changes for macOS for both the React libs and for RNTester.